### PR TITLE
Log workaround

### PR DESCRIPTION
**Pull request by jamiebrynes7**
Adds a workaround so we can access logs without pinging EV. If there is an error, we cat the log file if the env variable is defined. 

Will test locally before merging

### CODE REVIEW

File: scripts/build-worker.sh [hunk 1]
```diff
@@ -5,6 +5,16 @@ if [[ -n "${DEBUG-}" ]]; then
   set -x
 fi
 
+# Workaround until the artifact proxying service is set up properly.
```
Comment (jared-improbable) [inline scripts/build-worker.sh:8]: Link to ticket? :)

File: scripts/build-worker.sh [hunk 2]
```diff
@@ -5,6 +5,16 @@ if [[ -n "${DEBUG-}" ]]; then
...
+# At least this gives us terminal output
+function printLog() {
+    if [[ ${LOG_LOCATION-} ]]; then
+        cat "${LOG_LOCATION}"
```
Comment (jared-improbable) [inline scripts/build-worker.sh]: This may be over-thinking, but would it be better to send to `stderr`?

**Review by jared-improbable — APPROVED** [review]

File: scripts/build-worker.sh (hunk 2 above)
Comment (jamiebrynes7) [reply scripts/build-worker.sh]: Probably for correctness yes

File: scripts/build-worker.sh (hunk 1 above)
Comment (jamiebrynes7) [reply scripts/build-worker.sh:8]: Will have a dig around and see if I can uncover it 😉

**Pull request merged**